### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by zsigmond-czine-everit**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,5 +4,9 @@ jdk:
 before_script:
```
Comment (attila-kiss-it) [inline .travis.yml]: Please remove the before_script section, this project is not related to databases.

Comment (zsigmond-czine-everit) [inline .travis.yml]: Done.
